docs/blog/20221112-Daeraxa-ExamplePost.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Example Post
+author: Daeraxa
+date: 2022-11-12
+category:
+  - website
+  - testing
+tag:
+  - new feature
+  - placeholder
+sticky: false
+star: false
+article: false
+---
+
+This is the excerpt.
+
+<!-- more -->
+
+This is the body of the article.
+
+![image](./assets/test.png)
```

Comment:
I love that you made sure to include this

Reply:
How could I possibly resist?